packer/templates/windows_2008_r2.json

```diff
@@ -131,7 +131,7 @@
         "{{user `resources_dir`}}/certs/microsoft_root_2011.cer",
         "{{user `resources_dir`}}/certs/thawte_primary_root.cer",
         "{{user `resources_dir`}}/certs/utn-userfirst.cer",
-        "./resources/drivers/virtio/*"
+        "./resources/drivers/virtio/virtio-win-0.1.117_amd64/amd64/Win2008R2/*"
```

Comment:
This hardcodes a value extracted as a variable in your change to `virtio-win-drivers.sh`, since the script already has to extract the files it would be better to leave this as is and have the script setup the path to conform to the build environment needs.

Reply:
Reverted